backend/open_webui/routers/users.py

```
@@ -39,7 +48,16 @@ async def get_users(
     user=Depends(get_admin_user),
 ):
-    return Users.get_users(skip, limit)
+    users: List[UserModel] = Users.get_users(skip, limit)
+    credit_map = {
+        credit.user_id: {"credit": "%.4f" % credit.credit}
+        for credit in Credits.list_credits_by_user_id(
+            user_ids=(user.id for user in users)
+        )
+    }
+    for user in users:
+        setattr(user, "credit", credit_map.get(user.id, {}).get("credit", 0))
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)